demisto_sdk/commands/upload/uploader.py

```diff
@@ -677,14 +683,16 @@ def is_uploadable_dir(path: Path) -> bool:
 
 
 def parse_uploaded_successfully(
-    content_item: Union[Pack, ContentItem], zip: bool
+    content_item: Union[Pack, ContentItem], zip: bool, tpb: bool
 ) -> Iterable[Union[Pack, ContentItem]]:
     # packs uploaded unzipped are uploaded item by item, we have to extract the item details here
     if isinstance(content_item, Pack) and not zip:
         return iter(
             filter(
-                lambda content_item: content_item.content_type
-                not in CONTENT_TYPES_EXCLUDED_FROM_UPLOAD,
+                lambda content_item: (
+                    content_item.content_type not in CONTENT_TYPES_EXCLUDED_FROM_UPLOAD
+                    or (tpb and content_item.content_type == ContentType.TEST_PLAYBOOK)
+                ),
```

Comment:
```suggestion
                lambda content_item: (
                    content_item.content_type not in (CONTENT_TYPES_EXCLUDED_FROM_UPLOAD + ContentType.TEST_PLAYBOOK)
                ),
```
make sure that syntax works